DistributionSystemApi/DistributionSystemApi/Services/RecipientGroupService.cs

```diff
@@ -0,0 +1,156 @@
+﻿using DistributionSystemApi.Data.Entities;
+using DistributionSystemApi.Data;
+using Microsoft.EntityFrameworkCore;
+using DistributionSystemApi.Requests;
+using DistributionSystemApi.Responses;
+
+namespace DistributionSystemApi.Services
+{
+    public class RecipientGroupService
+    {
+        private readonly ContentDistributionSystemContext _context;
+
+        public RecipientGroupService(ContentDistributionSystemContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<RecipientGroupResponse>> GetRecipientGroups(CancellationToken cancellationToken)
```

Comment:
Services should not know about types which api provides as contracts. They should use/return their own types